training/general/src/environments/pytorch-1.11-cuda11.3/context/Dockerfile

```diff
@@ -2,26 +2,8 @@ FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{lat
 
 
 # Install pip dependencies
-RUN pip install 'ipykernel~=6.20.2' \
-                'azureml-core=={{latest-pypi-version}}' \
-				'azureml-dataset-runtime=={{latest-pypi-version}}' \
-                'azureml-defaults=={{latest-pypi-version}}' \
-				'azure-ml=={{latest-pypi-version}}' \
-				'azure-ml-component=={{latest-pypi-version}}' \
-                'azureml-mlflow=={{latest-pypi-version}}' \
-                'azureml-telemetry=={{latest-pypi-version}}' \
-		        'azureml-contrib-services=={{latest-pypi-version}}' \
-                'torch-tb-profiler~=0.4.0' \
-				'py-spy==0.3.12' \
-		        'debugpy~=1.6.3'
-
-RUN pip install \
-        azure-ai-ml==1.2.0 \
-        azureml-inference-server-http~=0.7.0 \
-        inference-schema~=1.4.2.1 \
-        MarkupSafe==2.0.1 \
-	    regex \
-	    pybind11
+COPY requirements.txt .
+RUN pip install -r requirements.txt
```

Comment:
Please also use `--no-cache-dir` when calling `pip` (applies to other envs too).